### PR TITLE
Add Tanya to student list

### PR DESCRIPTION
**Pull request by tretyak-pm**

### CODE REVIEW

**Comment by ava8katushka** [comment]
Please resolve the conflicts

File: README.md [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
+| 7 | Полина    | Карты. Деньги. Два ствола              |
+| 8 | Аня       | Начало                                 |
+| 9 | Дарья     | Крестный отец-10 :)                    |
+| 10| Наташа    | Реальная любовь                        |
+| 11| Вова      | Гладиатор                              |
+| 12| Анна      | Трасса 60                              |
+| 13| Виталя    | Матрица.                               |
+| 14| Ольга     | Скрытые фигуры                         |
+| 15| Юлия      | Пятый элемент                          | 
+| 16| Анастасия | Паразиты.                              |
```
Comment (ava8katushka) [inline README.md]: Where is your own addition?

**Comment by ava8katushka** [comment]
Yet conflicts :(. See your pull request in the course repo to resolve.

**Comment by ava8katushka** [comment]
Looks good to me, but for some reason git still thinks there are conflicts. Could you resolve it so it would show ready to merge? Seems like it's in the line next to last.
You can view your pull request in the course repo and try to resolve it directly from there.

**Comment by tretyak-pm** [comment]
Yes, I'm trying to understand where the conflict is coming from, as seems like versions match :/ will try again, maybe an extra space got in there somewhere...

**Comment by tretyak-pm** [comment]
@ava8katushka  seems to be resolved now!

**Pull request merged**